helm-swoop.el

```diff
@@ -685,6 +690,7 @@ If $linum is number, lines are separated by $linum"
                 (or $source
                     (helm-c-source-swoop))
                 :buffer helm-swoop-buffer
+                :display-function helm-swoop-split-window-function
```

Comment:
the display function is already set via `(let ((helm-display-function helm-swoop-split-window-function) ...` above, why add it again? same applies to the code below

Reply:
No real reason, it's lint from when I was trying to figure this out. I'll fix that :+1: thanks for letting me know!